Xamarin.Forms.Platform.Android/VisualElementPackager.cs

```diff
@@ -152,7 +153,23 @@ void EnsureChildOrder()
 				{
 					IVisualElementRenderer r = Platform.GetRenderer(element);
 					if (r != null)
+					{
 						(_renderer.View as ViewGroup)?.BringChildToFront(r.View);
+
+						if (Forms.IsLollipopOrNewer)
```

Comment:
Is this only broken in Lollipop or newer, too? Or will we still have this problem in 19 & 20?

Reply:
setElevation was only added in API 21 so pre 21 it's all just ignored
I ran those ui tests locally on API 19 and they all passed..